virttest/utils_test/libvirt.py

```diff
@@ -2555,8 +2555,17 @@ def set_vm_disk(vm, params, tmp_dir=None, test=None):
                                                               dist_img)
 
         # Mount the gluster disk and create the image.
-        process.run("mount -t glusterfs %s:%s /mnt; %s; umount /mnt"
-                    % (host_ip, vol_name, disk_cmd), shell=True)
+        #process.run("mount -t glusterfs %s:%s /mnt; %s; umount /mnt"
+        #            % (host_ip, vol_name, disk_cmd), shell=True)
+        ret = process.run("mount -t glusterfs %s:%s /mnt; %s"
```

Comment:
@dzhengfy How about adding a step to clear the dirs before creating a disk?
for instance, change the command lines as below:
1. mount glusterfs to /mnt
2. remove unused files/dirs under /mnt
3. qemu-img convert
4. check the result of qemu-img cmd